man/systemd-stub.xml

```diff
@@ -142,11 +144,27 @@
       details on system extension images. The generated <command>cpio</command> archive containing these
       system extension images is measured into TPM PCR 13 (if a TPM is present).</para></listitem>
 
+      <listitem><para>Similarly, files
+      <filename><replaceable>foo</replaceable>.efi.extra.d/*.addon.efi</filename>
+      are loaded and verified as PE binaries, and a <literal>.cmdline</literal> section is parsed from them.
+      In case Secure Boot is enabled, these files will be validated using keys in UEFI DB or Shim's MOK, and
```

Comment:
These would be validated via UEFI DB, **Shim's DB** or Shim's MOK, right?

Reply:
What do you mean by Shim's DB?

Reply:
shim's builtin DB (from [.vendor_cert](https://github.com/rhboot/shim/blob/0601f44760fb10550422090cc2761065047f56ab/cert.S#L24) section)

Reply:
Well the roles involved are different.
UEFI DB is mostly Microsoft or your hardware vendor
Shim DB is Canonical or RedHat 
MOK is somewhat "end user" / system owner.

Those don't *have* to be the case, but i think its good to show that 3 different entitites can sign these extensions.